evidence_tab.php

```diff
@@ -71,9 +75,9 @@ function evidence_display_form() {
 	$host_id = get_filter_request_var('host_id');
 	$template_id = get_filter_request_var('template_id');
 	$scan_date = get_filter_request_var('scan_date', FILTER_VALIDATE_REGEXP, array('options' => array('regexp' => '/^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}$/', 'default' => -1)));
-	$find_text = get_filter_request_var('find_text', FILTER_VALIDATE_REGEXP, array('options' => array('regexp' => '/[^[:print:]\r\n]/', 'default' => '')));
+	$find_text = get_filter_request_var('find_text', FILTER_VALIDATE_REGEXP, array('options' => array('regexp' => '/^([a-zA-Z0-9_\-\.:\+ ]+)$/', 'default' => 'INCORRECT: ' . get_nfilter_request_var('find_text'))));
 
-	print '<form name="form_evidence" action="evidence_tab.php">';
+	form_start(htmlspecialchars(basename($_SERVER['PHP_SELF'])), 'form_evidence');
```

Comment:
Should use `html_escape()` is my guess on this one.

Reply:
Fixed